### DISCUSSION TITLE
PAI Burin 手术 90 天后写下的手术常见问题 + 通用的手术准备清单

### DISCUSSION THREAD

**Discussion by Misaka-0x447f** [project-trans/basket-relay#4] [General]
> 手术记录(含大量敏感媒体)：https://t.me/+ry91g0QDl4diMWQ1
> 叠甲：本文有私货和大量个人经验内容，因此以下内容的提供均不负任何责任，不保证没有夸张、演绎成份，必须交叉参考更多人的经验。本人非利益相关。如果你不信任我或者任何人，我无法自证，但请记住，信源越多，判断越准，请交叉参考。最后，你杠就是你对。
> 允许转载。转载协议：[![CC BY-SA 4.0](https://licensebuttons.net/l/by-sa/4.0/88x31.png)](https://creativecommons.org/licenses/by-sa/4.0/)(知识共享-署名-相同协议共享)；推荐署名方式为带链接的姓名：[misaka17535](https://misaka.org/tg)

以下货币单位未注明均为中国大陆在岸人民币元，国内默认指中国大陆地区。
## 我能多大程度上信任这里的经验？
⚠️ 叠甲：每个人的自身情况不同，经验只能作为参考，如果你有任何状况强烈建议先咨询你的主治医师并以主治医师为准。  
不过，你的代理/中介可能会嫌麻烦或者出于利益原因不愿意让你见，态度可以强硬一点。  
## 手术的风险有哪些？
毫无疑问，最严重的风险是死亡，这种事情每年都会发生几例，原因不外乎主治医师或医院不负责、自己对自己不负责（术后马上放飞自我/耐痛导致大出血死亡；手术之后被中介介绍，跑到 kamol 去住院然后死亡而 kamol 对此人并不知情）、严重感染、基础病等。由于通常租借全科医院的手术室进行手术，麻醉当场死亡风险反而是最低的。
除此以外，还有可能出现例如阴唇缺损、下联合撕裂（这个常见）、排尿不畅或尿血、尿痛（本人第三个月开始出现，这是妇科由于尿道较男性缩短而出现的常见疾病，原生女性也会出现）、神经损伤（本人因术中操作不当左侧大腿神经损伤，目前六个月未完全恢复）等情况。
### 通模具和清洗
PAI Burin 的通模具要求最少，但是也要一天一次。通模具会剧痛，我本人是用利多卡因和送 5 秒、抽 10 秒的方式来克服的。越大的模具越痛。模具升号会重新开始痛，而且通完之后会出组织液需要一直使用卫生巾。如果是 WIH 的模具要求还有 6 个月不洗阴道的要求，**_只有约 2% 的 WIH 手术人群能坚持通模具半年_**，可参考 https://twitter.com/kawaiiRainMoe 的经历，术前觉得自己能一直通，术后 40 多天就摆烂了。但我个人仍然不建议做零深度。不要问我这个 2% 的数据是哪来的，样本量大概 100 多，可能含有演绎成份。如果手术后一个月就不通的话，阴道会不断萎缩直至零深度的状态，而且难以清洗。如果手术后六个月左右开始不通，一样会萎缩，也需要维持清洗。
而且只要通模具就要一直用卫生巾，有传统卫生巾和液体卫生巾可选，价格 1～3 元一片，
关于不同医院手术后六个月内通模具的要求，这里列出经验数据以供参考，不要找我要来源，医生说的就是对的。
WIH： 一天三通，一次一小时。
Kamol：一天两通，一次 50 分钟，算上清洗时间一次一个半小时，因为要兑碘液等。
PAI：前一个月一天两通，一个月到五个月一天一通，每次 15-30 分钟，通的次数比单次时间重要。我现在已经三五天才一通了，不建议学习。
  ### 疼痛
就算术后给你用吗啡止痛，出院之后还是会一直痛。本人现在 2 个月了，走路的时候还是有类似下垂感的疼痛。4 个月左右基本消失。有 Kamol 做手术的人会 6 个月还在神经抽痛。更何况有的人吗啡都没用…
另外尿路感染也会疼，可能会尿血，这点倒不是手术的问题，即使是原生女性也会频繁发生尿路感染。
  ### 独自住院
你需要独自住院，这可能是手术的最大的需求，医院不提供双人病房。也有很大可能在回到旅店后继续面临孤独，这是术后一个月自杀率升高的最主要的原因。请务必找人组队。对我个人而言 SteamDeck 和 HifiRush 这款游戏帮了很多。
  ### 女性要面对的生理问题
虽然不用面对月经，但是上文提到的尿路感染也是一个大问题。另外有一些疾病女性发病率要比男性高，比如甲状腺囊肿需要手术切除。没错，我就去做手术了。
此外，不要以为上厕所比以往更简单了。实际上如果有小便就需要用纸巾 + 湿巾擦尿道口，以防留下味道。
  ### 生殖器官灵敏度下降
手术可能导致灵敏度下降或无法正确将其与快感联系起来。本人是受害者。
## 我没有被劝退，手术要多少钱
在 PAI Burin 做皮瓣手术，在不出意外的情况下 10 万元(人民币)可以覆盖所有状况。除此以外的医院不会有 50% 以上的浮动。
如果出意外，请参考以下案例
（个人经验意外发生率大于 1% 小于 10%。）
手术后一个月快回国的时候由于排便过于用力撕裂并失血 3 升，送去 Phyathai 2 医院急救，花了 2 万元。
手术后脚部神经出问题，踝关节发炎，花了 1600 元。术后 3 个月后决定去切开粘连部分并植皮，再花 2 万元住院 15 天。（此案例缝合了两个案例，但时间和金钱成本是真实的）
选择 kamol 的腹膜技术(PAI 是没有腹膜的)，由于技术不成熟手术不太成功并导致大范围感染，在泰修养一年，花了百万元级别。
手术后需要回国修养，术后 2 个月内不能上班，包括还要找工作，对于一线城市如果没有收入来源回国生活费部分建议准备 5 万元以上。
## 选择哪种手术方式
我个人的经验是只考虑皮瓣。结肠、腹膜一是不成熟且价格昂贵，二是没有明显优势。但中介可能会给你推腹膜因为贵。你也别只信中介或者只信我，再多找几个信源去问问就好。
See also: [mtf.wiki 的相关文章](https://mtf.wiki/zh-cn/docs/srs/tech/)
## 深度预期
上限(初始深度)完全取决于你的身高，180 cm 是 7 英寸。下限取决于你的阴茎包皮、阴囊皮肤还有通模具的勤快程度还有是否植皮，本人初始 7 英寸，15 天就开始一天一通了，现在只有 4 英寸。
*后续有可能通回来。
不要觉得这么长没必要，因为你可能会不通，不通就会萎缩。
## 去泰国手术需要排队吗？
通常需要排队，可能要等几个月。需注意如果自己预约可能会预约到 PAI Sutin。PAI 是诊所名称，Sutin 是医生名称，擅长做欧美人的手术。
## 我还没开始排队，我该如何迈出第一步？
先去找一个人组队手术。如果不组队，你会在休养期间遇到各种各样的精神问题，如果没有人陪你你会疯。不要觉得自己扛得住。组队的过程或许会比排队更长。
然后别忘了同时办护照和签证。
## 手术之后需要多少天回国？
在 PAI Burin 进行手术后，由于复诊需求，最短，最短，最短 15 天可以回国，由于坐飞机的过程很痛苦建议 3 周以上。急的话，谨遵医嘱，一天两升水，能让你好的快点。
## 手术时间线
以下时间线为 PAI Burin 的，乐观估计，仅供参考。
`T- 30` 停用激素类药物
`T-  5` 建议落地时间，可以在充分考虑当地天气的前提下去旅游
`T-  3` 初诊，备皮
`T-  1` 喝泻药，做手术准备
`T+  0` 术后吃地西泮，减少肌肉紧张情况，会导致头晕
`T+  2` 尝试下床走路
`T+  4` 拆包、允许洗澡，但是可以再等一天
`T+  5` 出院回旅店，你就只交了这么多钱
`T+  7` 复诊，第一次尝试拔尿管，连续排尿 3 次
`T+ 15` 复诊，如果上次不成功再次尝试拔尿管，顺利的话出文件
`T+ 15` 最短离境时间，可以尝试 26mm + 30mm 模具联合使用
`T+ 30` 按个人情况可以尝试30mm + 32mm 模具联合使用
`T+ 30` 谨慎尝试坐班
`T+ 45` 疼痛缓解，停止使用抗生素
`T+ 60` 部分人可以坐班，谨慎尝试刺激阴唇
`T+ 90` 谨慎尝试性交和停止使用卫生巾
`T+180` 允许尝试游泳，功能基本恢复
## 我该选中介还是自己联系医生？
首先，泰国的这几家都存在中介返点，因此中介价格一般和诊所一样或更低(某雨另说，返点超高的同时还涨价)；其次，就算你直接联系诊所，诊所可能也会把你的业务转交给合作密切的中介来降低沟通成本。对于 PAI，推荐联系中介。自行联系可能会约到 PAI Sutin。对于 Kamol，医院官方中文中介是最不负责任的，第三方中介可能负责任但是可能会手术头一晚涨价。以上均为道听途说，请交叉参考更多来源。
## 模具上的宽度标注准吗？会不会跨度太大？
不准，但影响不是那么大。![image](https://github.com/project-trans/basket-relay/assets/15797507/956cdc19-87ec-4849-8013-2e49f844e2c1)
本人额外获得了 Kamol 的模具，2 个月的时候是 26 > 28(kamol) > 30.5 三个模具联合使用。
## PAI 的红黄瓶国内怎么买？
黄瓶是聚维酮碘，泰国的等效碘浓度为1% w/v，在中国大陆买不到一样的，可以考虑这个，不过这个浓度是0.5% w/v，需要加倍兑：https://item.taobao.com/item.htm?id=645807534248
红瓶我个人认为是加了润滑的聚维酮碘。
另外个人推荐用这种定量糖浆瓶储存聚维酮碘，兑起来更方便(但是不要长期储存)：https://item.taobao.com/item.htm?id=630753143343
长期储存用实验室的棕色带刻度细口瓶即可。
btw 关于生理盐水，用娃哈哈就好了...
## 润滑好贵
~~https://item.taobao.com/item.htm?id=685160746788
来历不明，但50g4块钱，似乎是仿品
不要粘稠的话方案更多~~
更新：上面这个已经失效了，推荐买独立品牌名的仿品，关键字KVY润滑液

**Comment by Misaka-0x447f** [comment]
## 参考准备清单
### 文化差异
⚠️ 尽管泰国成人情趣产业横行，但泰国不允许携带任何成人情趣用品，一旦发现会面临最高 60000 泰铢的罚款及最高 3 年的监禁。

### 出发前
和别人组队。出院后的陪伴比住院期间重要。

预留 4 天左右到泰。

更推荐在医院附近找旅店，而不是诊所。虽然你复诊得去诊所，但是如果出了什么事还是得送医院。

只要预算充足，住院期间不要退掉旅店，这样就不需要出院时进行行李搬运长时间的等待。

#### 视个人情况回国前推荐网购买好备在家里的东西
- 生理盐水（可以买妆字，建议 1688，一箱起步）
  - 用娃哈哈代替也行
- 润滑液注射器（多买几个，容易丢）
- 塑料便盆（买金属的也行，我忘了诊所送不送）
- 安全套（1688，随随便便买一大堆。）
- 利多卡因凝胶
- 润滑液和优碘（看上面的 faq
- 护垫（推荐一次垫 2 张，防漏，一箱起步）
- 卫生巾（买超薄的，如果你足够有钱就买液体卫生巾）
- 定量泵头（用于定量添加优碘，可以不买，奶茶店用的那种就行，看上文）

#### 出国前待办事项
- 提前两三周去中国银行换钱
  - 官方要求带 10000 泰铢，但为了买药等等推荐带 15000 泰铢
- 手机卡开通国际漫游功能，现在的手机卡都是默认关闭的

行李箱里别塞能在泰国轻易买到的东西，比如护理垫、卫生巾等。

#### 行李箱里推荐放的泰国买不到的东西
- 娱乐设备
  - switch / steam deck / 甚至笔记本
  - 你还需要考虑手机支架
  - 针对笔记本，推荐床上小桌板。到泰国了就只能网购了。即便是网购也不要指望能按时送到。
    - 如果你迫不得已要网购，关键词是 bed table，可以在 shopee.co.th 购买。
- 插座转换器、充电器
  - 这东西可一点都不好买，你应该不会希望专程去一趟小米之家。
- 两套睡衣
  - 不要指望有烘干机
- _根据个人习惯，再评估一下有什么别的买不到的东西比如化妆品_

#### 行李箱里推荐放的泰国买得到的东西
- 纤维素胶囊/芦荟胶囊，防止排便困难导致大出血
- 抱枕
- 蛋白粉
  - 2 桶，术后立即开始饮用
- 电动搅拌杯，搅蛋白粉用

### 手术前

- 私运药品列表
  - 买些止痛药(曲马多)。
    - 国内没得卖。你或许会用到。不要多带，推荐 3-5 板，最多 10 板，量太大容易被当成毒品处理。
  - 买些诺坤复。
    - 体积小好携带，也容易方便地解释为个人用药。推荐带 100 盒，盒子一定要拆掉。
  - *不要*买泰补，一个价。
- 旅行注意
  - 如果实在没有目的地，可以去暹罗(Siam)站附近逛 Central World / Paragon / Siam center / MBK
  - 泰国治安没有太好，注意晚上 8 点前叫计程车返程，超过这个时间叫计程车会很麻烦
  - 肯德基等连锁店的口味与国内差异较大
  - 看见中式居酒屋的招牌的话，快跑！！！

你的术前最后一次出门就餐机会是在手术前一天的中午。

#### 购物清单：
- ℹ️ 上面推荐过的不再重复推荐  
- ℹ️ 消耗品备足 15-25 天 * 2 次通模具的量，因为你术后很长时间无法出门买东西，宁可多买也不少买
- 洗发水
- 沐浴露
- 纸巾(1 包/3 天)
- 润滑液(25g-50g/次)
- 卫生巾(3 个/天)
- 饮用水(3 升/天，强烈建议备足 20 天以上的饮用水，这东西又不贵，术后多喝水多喝水多喝水一天起码 2 升)
- 便利店盒饭(去医院前一天买好，至少 5 天的量)
- 应急长保食物(防止盒饭不够吃又不能出门)
- 生理盐水(500ml/次)
- 棉签棉球
- 安全套(2 个/次)
- 利多卡因凝胶(2 管，止痛)
#### 可选购物清单：
- 祛疤膏 Dermatrix(出院后再用，0.2 支/天，备足 60 天，回国还要用)
- 洗手液
- 少量洗衣液
- 湿巾(住院用)
- 零食
- 电热蚊香
- 拖鞋
- 如果你是在 Phyathai 2 住院，记得拿送的爽身粉

### 手术后
多喝水，一天 2 升起，最好 3 升，消肿快，不消肿的后果包括但不限于无法按时拔尿管按时回国

喝蛋白粉，方便恢复

7 天和 14 天各有机会出门一次，记得采购盒饭

你可以在诊所买到彩色模具，600 元人民币一套，纪念意义为主

通模具次数比单次时长重要的多

排尿排便不要用力，强烈推荐提前买好芦荟胶囊，防止大出血

回国的时候能少折腾就少折腾，不要考虑转机去别的什么地方看朋友

机场可以提供轮椅服务，但是需要预约，记得提前 3-4 天咨询你的机票提供商，你可以在值机柜台开始就要求轮椅服务

vip 休息室比头等舱划算的多

**Reply by chika725** [reply]
可以问一下聚维酮碘加倍兑大概是多少杯

**Reply by name4hidden** [reply]
红霉素软膏可以处理一些微小的伤口，推荐携带，可配合利多卡因凝胶使用。

PAI 诊所可以在 `T+7` 复诊日提供纤维素帮助通便，但我并未使用。

爽身粉对干性皮肤有反作用（瘙痒干燥等），是个人情况可以使用身体乳。

疤痕膏可以尽早使用，我个人在 `T+14` 使用时效果相对一般，可能第一次复诊后即可使用。

**Comment by Z-Siqi** [comment]
**关于疼痛的个案补充：**
我曾在 6月7日，2024年 由 PAI 的 Burin 进行了SRS皮瓣手术。我个人感到的疼痛总体不多，仅在术后苏醒时伴有疼痛；在每小时 10ml 的麻烦止痛下到出院为之完全不痛。在出院后的第一周，除了坐着会感觉有些胀痛以外几乎没有不适，走路这些完全正常没有不适。一周以后则是坐着的疼痛缓解并在术后大约第三周离开泰国。总体而言对我而言的最痛苦的时候是刚拔尿管首次小便时最为难受以及困难。

**截至SRS时我的一些基本信息**
年龄：18
抽烟：从不
喝酒：从不
身体状况：正常

**我所见过的其她SRS后的人**
她们似乎恢复状况普遍没我好，有的是站直会疼痛，还有走路会有不适的人。而在这些人之中很多都存在抽烟或喝酒历史。因此，我大概率属于个案。但考虑到其它对于烟酒等影响的研究，这仍然可以支持 不抽烟不喝酒以及较小的年龄 有利于降低在围手术期内承受的痛苦。

**Reply by Misaka-0x447f** [reply]
感谢贡献！

**Reply by BishopWhite** [reply]
所有的外科手术都要求禁烟限酒，因为吸烟对血管内膜有损伤，极不利于恢复。所有手术想要恢复好最起码术后一周不能吸烟，最好术前一周也不要吸烟。

**Comment by NotEvenANeko** [comment]
红瓶应该是 7.5% 浓度也就是 0.75% w/v 等效碘浓度的聚维酮碘，瓶身说明貌似并没有写能润滑但是确实医生是拿这东西润滑的…

安睡裤比卫生巾更舒服

**Reply by name4hidden** [reply]
夜安裤确实更舒服，即便在刚刚出院也是如此。但Burin建议在酒店歇息期间尽量不穿着夜安裤或卫生巾以让患处接触空气，可以加速愈合。视个人情况而定，我更倾向于活动使用一次性内裤加卫生巾，卧床使用尿垫，睡眠使用夜安裤。

**Reply by clansty** [reply]
我刚术完那阵子不出去的时候都没穿内裤
睡觉就把狐狸垫垫在下面

**Comment by NaiveTomcat** [comment]
红色的碘是Betadine Surgical Scrub，设计用途是手术室刷手/术前皮肤消毒/平时手消，里面应该是7.5%的聚维酮碘和表面活性剂

[产品介绍页面](https://betadine.com/medical-professionals/betadine-surgical-scrub/)

内含表面活性剂确实能提供润滑但是接触开放创口会强烈灼烧感
